README.md

```diff
@@ -76,6 +76,7 @@ To use, register the service:
 ```yaml
 # config/packages/zenstruck_temp_file.yaml
 
-Zenstruck\TempFile\Bridge\Symfony\PurgeTempFiles:
-    tags: [kernel.reset]
+services:
+    Zenstruck\TempFile\Bridge\Symfony\PurgeTempFiles:
+        autoconfigure: true
```

Comment:
Or if you prefer explicit config we should add the method:
```suggestion
        tags:
            - { name: 'kernel.reset', method: 'reset' }
```

Reply:
Let's leave it as autoconfigure.